scripts/quests/sandoria/DRG_AF1_A_Craftmans_Work.lua

```diff
@@ -0,0 +1,133 @@
+-----------------------------------
+-- A Craftmans Work
+-- !addquest 0 94
+-----------------------------------
+-- Miaux !gotoid 17723442
+-- Easterrn Altepa ??? !gotoid 17244622
+-----------------------------------
+require('scripts/globals/items')
+require("scripts/globals/keyitems")
+require('scripts/globals/npc_util')
+require('scripts/globals/quests')
+require("scripts/globals/status")
+require('scripts/globals/interaction/quest')
+-----------------------------------
+local easternAltepaID = require("scripts/zones/Eastern_Altepa_Desert/IDs")
+-----------------------------------
+
+local quest = Quest:new(xi.quest.log_id.SANDORIA, xi.quest.id.sandoria.A_CRAFTSMAN_S_WORK)
+
+quest.reward =
+{
+    item     = xi.items.PEREGRINE,
+    fame     = 20,
+    fameArea = xi.quest.fame_area.SANDORIA,
+}
+
+quest.sections =
+{
+    -- Section: Quest available.
+    {
+        check = function(player, status, vars)
+            return status == QUEST_AVAILABLE and
+                player:getMainJob() == xi.job.DRG and
+                player:getMainLvl() >= xi.settings.main.AF1_QUEST_LEVEL
+        end,
+
+        [xi.zone.NORTHERN_SAN_DORIA] =
+        {
+            ['Miaux'] =
+            {
+                onTrigger = function(player, npc)
+                    if player:getVar("Quest[0][94]csSeen") == 0 then
+                        return quest:progressEvent(73)
+                    elseif player:getVar("Quest[0][94]csSeen") == 1 then --Declined First Offering.
```

Comment:
I think you could just use `quest:getVar('csSeen')` here since we are checking for a varable for this very quest.

Actually, I dont think `player:getVar` would work, it would be `player:getCharVar`

Reply:
xi.quest.getVar(ares, quest, 'name')

Reply:
good point - i was under the impression when I started this that it wouldnt call them but made the updates.